backend/presentation/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+
+from presentation.models import Presentation, Tag
+
+
+class TagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tag
+        fields = ["name"]
+
```

Comment:
If this serializer class will be used for read-only purposes, you could add `read_only_fields = fields` attribute to `Meta` class. Otherwise, keep it as it is.